docs/topics/Permissions.md

```diff
@@ -26,41 +26,44 @@ Below is a table of all current permissions, their integer values in hexadecimal
-| SPEAK                 | `0x000200000` | Allows for speaking in a voice channel                                                                                             | V            |
-| MUTE_MEMBERS          | `0x000400000` | Allows for muting members in a voice channel                                                                                       | V, S         |
-| DEAFEN_MEMBERS        | `0x000800000` | Allows for deafening of members in a voice channel                                                                                 | V, S         |
-| MOVE_MEMBERS          | `0x001000000` | Allows for moving of members between voice channels                                                                                | V, S         |
-| USE_VAD               | `0x002000000` | Allows for using voice-activity-detection in a voice channel                                                                       | V, S         |
-| CHANGE_NICKNAME       | `0x004000000` | Allows for modification of own nickname                                                                                            |              |
-| MANAGE_NICKNAMES      | `0x008000000` | Allows for modification of other users nicknames                                                                                   |              |
-| MANAGE_ROLES \*       | `0x010000000` | Allows management and editing of roles                                                                                             | T, V, S      |
-| MANAGE_WEBHOOKS \*    | `0x020000000` | Allows management and editing of webhooks                                                                                          | T            |
-| MANAGE_EMOJIS \*      | `0x040000000` | Allows management and editing of emojis                                                                                            |              |
-| USE_SLASH_COMMANDS    | `0x080000000` | Allows members to use slash commands in text channels                                                                              | T            |
-| REQUEST_TO_SPEAK      | `0x100000000` | Allows for requesting to speak in stage channels. (_This permission is under active development and may be changed or removed._)   | S            |
+| Permission            | Value                      | Description                                                                                                                        | Channel Type |
+| --------------------- | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| CREATE_INSTANT_INVITE | `0x0000000001` `(1 << 00)` | Allows creation of instant invites                                                                                                 | T, V, S      |
```

Comment:
any reason why they're always 2 digits instead of like `1 << 0`?

Reply:
Oh I just figured it was nice to have them line up?
didn't see any other places though

Reply:
tbh not even sure why permissions has the hex values, for the flag tables its just the shift:
* https://discord.com/developers/docs/resources/channel#message-object-message-flags
* https://discord.com/developers/docs/resources/guild#guild-object-system-channel-flags
* https://discord.com/developers/docs/resources/user#user-object-user-flags
* https://discord.com/developers/docs/topics/oauth2#application-application-flags

Reply:
> tbh not even sure why permissions has the hex values, for the flag tables its just the shift:

(Also intents.) I meant to bring this up at some point. Permissions hex values came first with the original docs. I think that documenting flags with their bit shifts was a trend started by Mason (#754) and it stuck. I wouldn't mind changing permissions to bit shifts, but I think there is familiarity with seeing them as their hex values (for example, people tend to recognize the admin permission as 0x8). Do you think we should have a column for both the bit shift and the hex value everywhere?

Reply:
afaik people recognize admin as 8 more than 0x8